charts/kubewarden-controller/chart-values.yaml

```diff
@@ -82,3 +90,27 @@ resources:
     requests:
       cpu: 250m
       memory: 50Mi
+  auditScanner:
+    limits:
+      cpu: 500m
+      memory: 50Mi
+    requests:
+      cpu: 250m
+      memory: 50Mi
+
+# experimental Kubewarden features. These features are not recommended for
+# production yet, and are not covered by SemVer guarantees.
+experimental:
+  auditScanner:
+    enable: true
```

Comment:
Given this is a PR against a feature branch I think it's fine to leave the scanner enabled by default